.ci/magic-modules/generate-terraform-all-platforms.sh

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env bash
+
+set -e
+
+# Setup GOPATH
+export GOPATH=${PWD}/go
+# Setup GOBIN
+export GOBIN=${PWD}/dist
+
+set -x
+
+# Create GOBIN folder
+mkdir -p "$GOBIN"
+
+# Create GOPATH structure
+mkdir -p "${GOPATH}/src/github.com/terraform-providers"
+yes | cp -rf "$1" "${GOPATH}/src/github.com/terraform-providers/"
```

Comment:
`yes |` and `-f` should be redundant, right?

Reply:
Yeah, I've tested it on my local machine several times and I wanted to avoid the overwrite prompt, but I think concourse create a container every time, so it's not necessary.

Reply:
Changed anyway